src/main/java/bsise/server/report/daily/service/DailyReportService.java

```diff
@@ -119,6 +123,66 @@ public DailyReportResponseDto getDailyReport(UUID userId, LocalDate targetDate)
         return DailyReportResponseDto.of(dailyReport, letterAnalyses);
     }
 
+    public void createDailyReportsBy(UUID userId, LocalDate startDate, LocalDate endDate) {
```

Comment:
```suggestion
    public List<DailyReport> createDailyReportsBy(UUID userId, LocalDate startDate, LocalDate endDate) {
```

생성한 데일리 리포트들을 반환하면 아래 WeeklyReportService에서 재사용 할 수 있습니다